packages/scandipwa/src/component/CheckoutShipping/CheckoutShipping.component.js

```diff
@@ -56,14 +79,15 @@ export class CheckoutShipping extends PureComponent {
         const orderTotal = formatPrice(grand_total, quote_currency_code);
 
         return (
-            <div block="Checkout" elem="OrderTotal">
-                <span>
+            <dl block="Checkout" elem="OrderTotal">
+                <dt>
                     { __('Order total:') }
-                </span>
-                <span>
+                </dt>
+                <dt>
                     { orderTotal }
-                </span>
-            </div>
+                    { this.renderOrderTotalExlTax() }
+                </dt>
+            </dl>
```

Comment:
Same here as in billign

Reply:
Also, is there any way to unify this logic? Worth investigating. Let's keep this DRY.

Reply:
Unified logic in Checkout container. Now shipping and billing uses the same.